Cartfile

```diff
@@ -1,4 +1,5 @@
 github "pusher/pusher-platform-swift" ~> 0.6.2
+github "pusher/push-notifications-swift" "users-implementation"
```

Comment:
This will probably change.

Reply:
@hamchapman I've temporarily changed the Carthage command setting on our CI from `bootstrap` to `update` in order to prevent caching - there was few changes in `pusher/push-notifications-swift`. Just to let you know. I'll change it back once we merge this PR.

Reply:
This was reverted back to how it was ^^.